zone/worldserver.cpp

```diff
@@ -1710,6 +1710,31 @@ void WorldServer::HandleMessage(uint16 opcode, const EQ::Net::Packet &p)
 
 	case ServerOP_ReloadTitles:
 	{
+		if (zone) {
+			worldserver.SendEmoteMessage(
```

Comment:
Lots of messages like this across a lot of the PR's. could be simplified into a method and save 23 lines each call.

eg `zone->NotifyWorldReloaded(std::string reload_label);`

Reply:
👍